### PR TITLE
Add sphinx docs building instructions to README.md

### LINKED ISSUE #2453: cannot build sphinx document
I am trying to build the documentation sphinx document but I cannot make it compile. The error I get is (jwst_dev) bash-3.2$ make html
sphinx-build -b html -d _build/doctrees   . _build/html
Running Sphinx v1.7.6

Configuration error:
There is a programable error in your configuration file:

Traceback (most recent call last):
  File "/Users/rdiaz/anaconda3/envs/astroconda/lib/python3.5/site-packages/sphinx/config.py", line 161, in __init__
    execfile_(filename, config)
  File "/Users/rdiaz/anaconda3/envs/astroconda/lib/python3.5/site-packages/sphinx/util/pycompat.py", line 150, in execfile_
    exec_(code, _globals)
  File "conf.py", line 19, in <module>
    import stsci_rtd_theme
ImportError: No module named 'stsci_rtd_theme'

Makefile:49: recipe for target 'html' failed
make: *** [html] Error 2
(jwst_dev) bash-3.2$ 

I downloaded a version of the stsci_rtd_theme package and installed it but still get the same error. Can someone help with this? I have spent already a lot of time trying to make it work.

### PR DESCRIPTION
**Pull request by jdavies-st**
Resolves #2453.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,6 +69,11 @@ Documentation (built daily from `master`) is available here:
 
 https://jwst-pipeline.readthedocs.io/en/latest/
 
+One can clone this repository and build the documentation with
+
+    pip install stsci_rtd_theme
```
Comment (sosey) [inline README.md]: They may also need to pip install `sphinx_rtd_theme` and  `sphinx-automodapi`

Comment (stscieisenhamer) [reply README.md]: 👍 on @sosey comment.

**Comment by jdavies-st** [comment]
Fixed.

File: README.md [hunk 2]
```diff
@@ -100,12 +105,11 @@ Software vs DMS build version map
 Unit Tests
 ----------
 
-Unit tests can be run via `pytest`.  All tests need a the `ci_watson` pytest plugin to run.  We also recommend using `pytest-xdist` so you can run them in parallel.
+Unit tests can be run via `pytest`.  All tests need a the `ci_watson` pytest plugin to run.
 
-    pip install ci-watson pytest-xdist
-    pytest -n <cores>
+    pip install requests_mock ci_watson
+    pytest jwst
```
Comment (jdavies-st) [inline README.md]: Removing the `pytest-xdist` example here, as this does not work for people who are not using the CRDS central store cache and need to get the files via the CRDS web service.

**Pull request merged**